pkgs/development/tools/testkube/default.nix

```
@@ -0,0 +1,31 @@
+{ buildGoModule, fetchFromGitHub, lib, stdenv }:
+
+buildGoModule rec {
+  pname = "testkube";
+  version = "1.10.40";
+
+  src = fetchFromGitHub {
+    owner = "kubeshop";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-6U9tbn1hp4OEI5ZNcknXBm64nJtZtC9nuIpsHSl1rQw=";
+  };
+
+  subPackages = [ "cmd/kubectl-testkube/main.go" ];
+
+  vendorSha256 = "sha256-DLh4uKT4/YbA0ZjEZXmryyB5BnlL2XD+8JvHy+Su6pM=";
+
+  ldflags = [ "-s" "-w" ];
```

```suggestion
  preConfigure = ''
    ldflags="-s -w -X main.version=${version} \
    -X main.commit="$(cat .git-revision)" \
    -X main.date="$(date +%Y-%m-%d)" \
    -X main.builtBy=nixpkgs"
  '';
```
Pass flags dynamically due to git-revision.